src/components/PatientProfilePage/HospitalTransportSelector.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Typography from '@material-ui/core/Typography';
+import { Hospital } from '../../graphql/queries/hospitals';
+
+const HospitalTransportSelector = ({
+  options,
+  currentValue,
+  handleChange,
+}: {
+  options: Array<Hospital>;
+  currentValue: string;
+  handleChange: (e: React.ChangeEvent<HTMLElement>) => void;
+}) => {
+  return (
+    <>
+      <Typography variant="h6" style={{ marginBottom: 20 }}>
+        Hospital
+      </Typography>
+      <RadioGroup
+        value={currentValue}
+        onChange={handleChange}
+        style={{ marginBottom: 32 }}
```

Comment:
I've a noticed several occurrences of non px values, do we want to be consistent?

Reply:
yes please, we should be sticking with explicit `px` values everywhere